src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireServiceTest.java

```diff
@@ -26,262 +26,281 @@
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+
+    when(proprietaireRepository.save(Mockito.any(Proprietaire.class))).thenReturn(proprietaire);
+
+    ProprietaireDTO savedProprietaire = proprietaireService.creerProprietaire(
+        proprietaireCreationDTO);
+
+    Assertions.assertThat(savedProprietaire).isNotNull();
+    Assertions.assertThat(savedProprietaire.getId()).isEqualTo(1L);
+    Assertions.assertThat(savedProprietaire.getNom()).isEqualTo(proprietaireCreationDTO.getNom());
+    Assertions.assertThat(savedProprietaire.getPrenom())
+        .isEqualTo(proprietaireCreationDTO.getPrenom());
+    Assertions.assertThat(savedProprietaire.getEmail())
+        .isEqualTo(proprietaireCreationDTO.getEmail());
+    Assertions.assertThat(savedProprietaire.getNumTel())
+        .isEqualTo(proprietaireCreationDTO.getNumTel());
+    Assertions.assertThat(savedProprietaire.getDateInscription()).isEqualTo(localDateTime);
+  }
+
+  @Test
+  public void ProprietaireService_GetProprietaire_ReturnsListProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+    Proprietaire proprietaire2 = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire2.setLogements(null);
+    proprietaire2.setId(2L);
+    when(proprietaireRepository.findAll()).thenReturn(List.of(proprietaire, proprietaire2));
+
+    List<ProprietaireDTO> proprietaireDTOS = proprietaireService.getProprietaire(false);
+
+    Assertions.assertThat(proprietaireDTOS).isNotNull();
+    Assertions.assertThat(proprietaireDTOS).hasSizeGreaterThan(1);
+
+  }
+
+  @Test
+  public void ProprietaireService_GetProprietaireById_ReturnOneProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+    when(proprietaireRepository.findById(proprietaire.getId())).thenReturn(
+        Optional.of(proprietaire));
+
+    ProprietaireDTO proprietaireDTO = proprietaireService.getProprietaireById(proprietaire.getId(),
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireDTO' must contain no more than '1' consecutive capital letters.